Adding mon, a perl based monitoring tool

sarlalian opened the pull request:
Adding mon, its super simple, but quite fast and reliable

Commit:
Adding mon, a perl based monitoring tool

Commit:
Adding Prometheus, a combined monitoring tool and time series database

Commit:
Adding Nanomsg, from the creator of ZeroMQ, a c based messaging api

n1trux commented:
Last update: 2013-07-23 – seems pretty dead. I wouldn't trust a monitoring solution which is still hosted on sourceforge ;)

sarlalian commented:
It is dead, but it is also dead simple, and works incredibly well.

n1trux commented:
If you find an active fork, I'd be happy to accept it into [my active fork of awesome-sysadmin](/n1trux/awesome-sysadmin).

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Add Arborist to the Monitoring Section.

Arborist is a monitoring toolkit for observing and reporting on a tree
of networked devices.  It is a framework for creating a bespoke
monitoring system that understands the dependency graph of your network.
